server/src/controllers/duplicate.controller.ts

```diff
@@ -15,4 +17,16 @@ export class DuplicateController {
   getAssetDuplicates(@Auth() auth: AuthDto): Promise<DuplicateResponseDto[]> {
     return this.service.getDuplicates(auth);
   }
+
+  @Delete()
+  @Authenticated()
+  deleteDuplicates(@Auth() auth: AuthDto, @Body() dto: BulkIdsDto): Promise<void> {
```

Comment:
Per the spec (https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.5) sending a body in a DELETE is kind of a "gray area". There is no guarantee that it's supported and it's more of like a "clients and servers may implement it".  It's probably fine for us if nest accepts it, just pointing out that it's maybe not ideal

Reply:
Sure, but this isn't the first one. We have a bulk delete assets endpoint that accepts a body as well.